backend/src/main/kotlin/org/loculus/backend/controller/SubmissionController.kt

```diff
@@ -76,33 +77,45 @@ open class SubmissionController(
     private val submissionDatabaseService: SubmissionDatabaseService,
     private val iteratorStreamer: IteratorStreamer,
     private val requestIdContext: RequestIdContext,
+    private val backendConfig: BackendConfig,
 ) {
-
     @Operation(description = SUBMIT_DESCRIPTION)
     @ApiResponse(responseCode = "200", description = SUBMIT_RESPONSE_DESCRIPTION)
+    @ApiResponse(responseCode = "400", description = SUBMIT_ERROR_RESPONSE)
     @PostMapping("/submit", consumes = ["multipart/form-data"])
     fun submit(
         @PathVariable @Valid organism: Organism,
         @HiddenParam authenticatedUser: AuthenticatedUser,
         @Parameter(description = GROUP_ID_DESCRIPTION) @RequestParam groupId: Int,
         @Parameter(description = METADATA_FILE_DESCRIPTION) @RequestParam metadataFile: MultipartFile,
         @Parameter(description = SEQUENCE_FILE_DESCRIPTION) @RequestParam sequenceFile: MultipartFile?,
-        @Parameter(description = "Data Use terms under which data is released.") @RequestParam dataUseTermsType:
-        DataUseTermsType,
+        @Parameter(
+            description =
+            "Data Use terms under which data is released. Mandatory when data use terms are enabled for this Instance.",
+        ) @RequestParam dataUseTermsType: DataUseTermsType?,
         @Parameter(
             description =
             "Mandatory when data use terms are set to 'RESTRICTED'." +
                 " It is the date when the sequence entries will become 'OPEN'." +
                 " Format: YYYY-MM-DD",
         ) @RequestParam restrictedUntil: String?,
     ): List<SubmissionIdMapping> {
+        var dataUseTermsKind = DataUseTermsType.OPEN
+        if (backendConfig.dataUseTerms.enabled) {
+            if (dataUseTermsType == null) {
+                throw BadRequestException("the 'dataUseTermsType' needs to be provided.")
+            } else {
+                dataUseTermsKind = dataUseTermsType
+            }
+        }
```

Comment:
The variable name `dataUseTermsKind` confused me a bit, first I thought that it would be something fundamentally/type-wise different than `dataUseTermsType`, but I'm not entirely sure what a better name would be though.. So no changes needed if no one has an idea for a better name.

Reply:
Maybe it could just be `_dataUseTermsType` so it's more clear that it's basically identical, just an _internal_ thing? I can totally see that it might be confusing.

Reply:
Turns out the linter says that variable names need to start with a lowercase letter, so I went with `innerDataUseTermsType`: https://github.com/loculus-project/loculus/pull/3468/commits/f838965c462d523681b445924144814f0186e0c7